cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/strategy/PostponeUrpReplicaMovementStrategy.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2018 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
+ */
+
+package com.linkedin.kafka.cruisecontrol.executor.strategy;
+
+import com.linkedin.kafka.cruisecontrol.executor.ExecutionTask;
+import java.util.Comparator;
+import org.apache.kafka.common.Cluster;
+
+import static com.linkedin.kafka.cruisecontrol.KafkaCruiseControlUtils.isPartitionUnderReplicated;
+
+/**
+ * The strategy which tries to first move replicas of partitions which currently are not under replicated.
```

Comment:
Nit: `The strategy, which tries to first move replicas of partitions that are currently not under replicated.`